provider-service/base/pkg/streams/sinks/webhook_sink_unit_test.go

```diff
@@ -0,0 +1,91 @@
+//go:build unit
+
+package sinks
+
+import (
+	"context"
+	"fmt"
+	"github.com/go-logr/logr"
+	resty "github.com/go-resty/resty/v2"
+	"github.com/jarcoal/httpmock"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/sky-uk/kfp-operator/argo/common"
+	. "github.com/sky-uk/kfp-operator/provider-service/base/pkg"
+	"go.uber.org/zap/zapcore"
+	"testing"
+)
+
+func TestSinksUnitSuite(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Sinks Unit Suite")
+}
+
+var _ = Context("SendEvents", func() {
+	var logger, _ = common.NewLogger(zapcore.DebugLevel)
+	var ctx = logr.NewContext(context.Background(), logger)
+	runCompletionEventData := common.RunCompletionEventData{
```

Comment:
Could default to zero values?

Reply:
I assume you mean just do `runCompletionEventData := common.RunCompletionEventData{}`, have changed.